Jenkinsfile

```diff
@@ -7,13 +7,14 @@ pipeline {
     }
     stages {
         stage('Build') {
+            agent { docker { image "python:3" } }
             steps {
                 echo 'Building..'
                 sh '''
-                    virtualenv ./sdk-deploy
+                    rm -r dist
+                    python3 -m venv ./sdk-deploy
                     . ./sdk-deploy/bin/activate
-                    python setup.py sdist
-                    python setup.py bdist_wheel
+                    python setup.py sdist bdist_wheel
```

Comment:
Is there a reason you use python some places and python3 others?

Reply:
Thanks. I had left one python3 accidentally from my testing.

I removed it since we are using a python 3 docker image.